yb-voyager/cmd/common.go

```diff
@@ -189,45 +189,122 @@ func printExportedRowCount(exportedRowCount map[string]int64, useDebezium bool)
+						headerfmt("INSERTS"), headerfmt("UPDATES"), headerfmt("DELETES"),
+						headerfmt("FINAL ROW COUNT(SNAPSHOT + CHANGES)"))
+
+				}
+				totalChangesEvents, inserts, updates, deletes, err := metaDB.GetExportedEventsStatsForTable(tableStatus.SchemaName, tableStatus.TableName)
+				if err != nil {
+					utils.ErrExit("could not fetch table stats from meta DB: %w", err)
+				}
+				table.AddRow(tableStatus.SchemaName, tableStatus.TableName, tableStatus.ExportedRowCountSnapshot, totalChangesEvents,
+					inserts, updates, deletes, tableStatus.ExportedRowCountSnapshot+inserts-deletes)
+
 			}
 		}
+
 	} else {
 		table.AddRow(headerfmt("TABLE"), headerfmt("ROW COUNT"))
 		sort.Strings(keys)
 		for _, key := range keys {
 			table.AddRow(key, exportedRowCount[key])
 		}
 	}
-
 	fmt.Print("\n")
 	fmt.Println(table)
 	fmt.Print("\n")
 
 }
 
+func printImportedRowCount(tasks []*ImportFileTask) {
+	tableList := importFileTasksToTableNames(tasks)
+
+	err := retrieveMigrationUUID(exportDir)
+	if err != nil {
+		utils.ErrExit("could not retrieve migration UUID: %w", err)
+	}
+
+	uitable := uitable.New()
+	headerfmt := color.New(color.FgGreen, color.Underline).SprintFunc()
+
+	snapshotRowCount := make(map[string]int64)
+	for _, tableName := range tableList {
+		tableRowCount, err := tdb.GetImportedSnapshotRowCountForTable(tableName)
+		if err != nil {
+			utils.ErrExit("could not fetch snapshot row count for table %q: %w", tableName, err)
+		}
+		snapshotRowCount[tableName] = tableRowCount
+	}
+
+	if useDebezium || changeStreamingIsEnabled(exportType) {
+		if changeStreamingIsEnabled(exportType) {
+			for i, tableName := range tableList {
+				if i == 0 {
+					uitable.AddRow(headerfmt("SCHEMA"), headerfmt("TABLE"), headerfmt("SNAPSHOT ROW COUNT"), headerfmt("TOTAL CHANGES EVENTS"),
+						headerfmt("INSERTS"), headerfmt("UPDATES"), headerfmt("DELETES"),
+						headerfmt("FINAL ROW COUNT(SNAPSHOT + CHANGES)"))
+				}
+				eventCounter, err := tdb.GetImportedEventsStatsForTable(tableName, migrationUUID)
+				if err != nil {
+					utils.ErrExit("could not fetch table stats from target db: %v", err)
+				}
+				uitable.AddRow(getTargetSchemaName(tableName), tableName, snapshotRowCount[tableName], eventCounter.TotalEvents,
+					eventCounter.NumInserts, eventCounter.NumUpdates, eventCounter.NumDeletes,
+					snapshotRowCount[tableName]+eventCounter.NumInserts-eventCounter.NumDeletes)
+			}
+		} else {
+			for i, tableName := range tableList {
+				if i == 0 {
+					uitable.AddRow(headerfmt("SCHEMA"), headerfmt("TABLE"), headerfmt("SNAPSHOT ROW COUNT"))
+				}
+				uitable.AddRow(getTargetSchemaName(tableName), tableName, snapshotRowCount[tableName])
+			}
+		}
+	} else {
+		for i, tableName := range tableList {
+			if i == 0 {
+				uitable.AddRow(headerfmt("SCHEMA"), headerfmt("TABLE"), headerfmt("ROW COUNT"))
+			}
+			uitable.AddRow(getTargetSchemaName(tableName), tableName, snapshotRowCount[tableName])
+		}
+	}
```

Comment:
Why are these two blocks identical? I guess we need only a single if..else as:
```
if change streaming is enabled {

} else {
  // snapshot-only import.
}
```

Reply:
yeah right